webpack.config.babel.js

```diff
@@ -91,7 +92,7 @@ module.exports = ({ stats = false } = {}) => ({
   plugins: [
     new HtmlWebPackPlugin({
       template: './src/html/index.html',
-      title: 'dFusion',
+      title: 'Gnosis Protocol',
```

Comment:
My understanding is that that for now we'll be using `dFusion` name and switch to `Gnosis Protocol` when we publicly launch.

@c3rnst

Reply:
I let @c3rnst answer that :)